.maint/contributors.json

```diff
@@ -199,5 +199,10 @@
     "affiliation": "State Key Laboratory of Cognitive Neuroscience and Learning, Beijing Normal University",
     "name": "Ye, Zhifang",
     "orcid": "0000-0003-0489-2619"
+  },
+  {
+    "affiliation": "Department of Radiology, University Hospital of Lausanne",
+    "name": "Provins, Céline",
```

Comment:
Names are ordered alphabetically (through the lastname), please move up to the right position.